ethstaker_deposit/settings.py

```diff
@@ -17,13 +17,17 @@ class BaseChainSetting(NamedTuple):
     GENESIS_FORK_VERSION: bytes
     EXIT_FORK_VERSION: bytes  # capella fork version for voluntary exits (EIP-7044)
     GENESIS_VALIDATORS_ROOT: Optional[bytes] = None
+    MULPLIER: int = 1
+    MINIMUM_COMPOUNDING_DEPOSIT: int = 1
```

Comment:
There is currently an issue with `MINIMUM_COMPOUNDING_DEPOSIT` being defined as an int and later on being used assigned a float in the Gnosis and Chiado setting. This was caught with the mypy run on https://github.com/eth-educators/ethstaker-deposit-cli/actions/runs/13702872871/job/38321047264 .

Ask yourself if this should also be a decimal or a float for fixed-precision arithmetic or if the lost in precision from floating point number arithmetic is fine for all use cases.